tutorials/CHT/flow-over-plate/buoyantPimpleFoam-laplacianFoam_nearest-projection/Allrun

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# This script prepares and runs all the participants in one terminal,
+# forwarding the solvers' output to log files.
+# Alternatively, you may execute the scripts "runSolid" and "runFluid"
+# in separate terminals.
+
+# The script "Allclean" cleans-up the result and log files.
+
+# Link to the precice-config for the serial run
+echo "Setting up the preCICE configuration file for a serial simulation..."
+ln -s -f nearest-projection-setup.xml precice-config.xml
+
+# Participant 1: Fluid
+Participant1="Fluid"
+Solver1="buoyantPimpleFoam"
+
+    # Prepare
+    echo "Preparing the ${Participant1} participant..."
+    blockMesh -case ${Participant1} > ${Participant1}_blockMesh.log 2>&1
+    checkMesh -case ${Participant1} > ${Participant1}_checkMesh.log 2>&1
+
+    # Run and get the process id
+    echo "Starting the ${Participant1} participant..."
+    ${Solver1} -case ${Participant1} |& tee ${Participant1}.log 2>&1 &
```

Comment:
This allows me to follow the simulation progress in the terminal and simultaneously save it in the log files without running additional commands. It depends on what you prefer.

Reply:
Then, this would be very useful in the `runFluid`-`runSolid` scripts! But the `Allrun` I would prefer to keep it like this, in order to have an overview of the complete simulation.

Reply:
Ok done.